src/api.coffee

```diff
@@ -16,7 +17,7 @@ privateAppEnvVars = [
 
 module.exports = (application) ->
 	api = express()
-	api.use(bodyParser())
```

Comment:
You could do something like:
```coffee

unparsedRouter = express.Router()
api.post '/v1/images/create', dockerUtils.createImage
...
parsedRouter = express.Router()
parsedRouter.use(bodyParser())
parsedRouter.post '/v1/update', ...
...
api.use(parsedRouter)
api.use(parsedRouter)
```
in order to separate routes that require body parsing and routes that don't.  It might even make sense to do something like
```coffee
unparsedRouter = express.Router()
api.post '/images/create', dockerUtils.createImage
app.use('/v1/local/', unparsedRouter)
```
to completely isolate it under its own path